### ISSUE TITLE
Expand on @service macro for giving service an alias

### ISSUE THREAD

**Issue by mattBrzezinski** [JuliaCloud/AWS.jl#435]
Stemming from a comment [here](https://github.com/JuliaCloud/AWSTools.jl/pull/27/files#r696159414) it would be nice if we could do something like;

```julia
@service cloudformation as CF
```

The problem originates from the `AWSTools.jl` repo, where there are sub-modules named `CloudFormation`. This becomes problematic with this package in naming them. As you need to use odd-casings to not have issues. e.g. `cloudFormation`.

**Comment by omus** [comment]
Makes sense to use the [`as` syntax](https://docs.julialang.org/en/v1/manual/modules/#Renaming-with-as) which is also used by `using`/`import`
